src/test/items/light_ball.test.ts

```diff
@@ -37,29 +37,29 @@ describe("Items - Light Ball", () => {
 
     const partyMember = game.scene.getParty()[0];
 
-    // Checking consoe log to make sure Light Ball is applied when getBattleStat (with the appropriate stat) is called
-    partyMember.getBattleStat(Stat.DEF);
+    // Checking consoe log to make sure Light Ball is applied when getEffectiveStat (with the appropriate stat) is called
```

Comment:
```suggestion
    // Checking console log to make sure Light Ball is applied when getEffectiveStat (with the appropriate stat) is called
```
NIT: Minor correction